70_FLIPFluids/app_resources/common.hlsl

```diff
@@ -17,13 +16,9 @@ static const float FLT_MAX = 3.402823466e+38;
 
 struct Particle
 {
-    float4 position;
-    float4 velocity;
-
-    uint id;
-    uint pad[3];
+    float32_t3 position;
+    float32_t3 velocity;
```

Comment:
SoA instead of AoS could still be used https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/145#discussion_r1796704304

@Erfan-Ahmadi maybe you're interested in benchmarking?

Reply:
nvm this struct is only used locally